### DISCUSSION TITLE
Option to hide / show shape in scene editor

### DISCUSSION THREAD

**Discussion by pixel-boy** [godotengine/godot-proposals#11387] [2D]
In the scene editor I often need to hide the shapes of the different elements.

It is feasible but very quickly tedious when there are many of them and they are sometimes in sub-scenes, which often discourages me from doing it, an option stored in "View" or a button icon next to the grid parameters could save time.

I use Godot for 2D mainly but it is possible that it also applies to 3D.

**Comment by Calinou** [comment]
https://github.com/godotengine/godot/pull/98370 addresses this for subscenes :slightly_smiling_face:
